gpio/gpioutil/debounce.go

```diff
@@ -53,6 +55,7 @@ func Debounce(p gpio.PinIO, denoise, debounce time.Duration, edge gpio.Edge) (gp
 		denoise:  denoise,
```

Comment:
Remove TODO at line 44?

[Optional] One option is to have an internal debounce() function that returns *debounced so then you don't have to cast in the unit test.

Reply:
Removed TODO. I don't think that a `debounce()` would feel cleaner than the cast, so leaving that as-is.